### ISSUE TITLE
Output of should.have.status() is very hard to read

### ISSUE THREAD

**Issue by rprieto** [shouldjs/http#1]
Hi,

When using `should.have.status()`, this is the test output I get:

``` js
unirest.get('...').end(function(res) {
  res.should.have.status(200);
  done();
});
```

``` bash
[.....]
Uncaught AssertionError: expected {
  _consuming: true,
  _dumped: false,
  _events: {

............ 12,000 lines later ..........

  status: 405,
  statusCode: 405,
  statusType: 4,
  toJSON: [Function: toJSON],
  trailers: {},
  unauthorized: false,
  upgrade: false,
  url: ''
} to have property statusCode of 200 (got 405)
```

It really is **12000** lines of serialised `response` object.
Would it be possible to simplify it to something like this?

``` bash
expected response to have property statusCode of 200 (got 405)
```

or maybe

``` bash
expected response {
  body: "in case there's something useful to debug"
} to have property statusCode of 200 (got 405)
```

Note: for comparison, the `should.have.header` matcher doesn't print the entire response, it only prints the headers, which is great for debugging.

(ps: I'm running `node @ 0.10.25` and `should-http @ 0.0.2`)

**Comment by btd** [comment]
Yes, both variants pretty easy to make. Just need to add `this.params = { actual: 'response' }` or `var act = { body: this.obj.body }; this.params = { actual: act }`.

**Comment by btd** [comment]
I just published 0.0.3, pls give it a try.

**Comment by rprieto** [comment]
Thanks a lot, it looks great!

``` bash
Uncaught AssertionError: expected {
  body: 'The response body',
  statusCode: 405
} to have property statusCode of 200 (got 405)
```
